robusta_krr/core/models/objects.py

```diff
@@ -13,12 +13,22 @@ def __hash__(self) -> int:
         return hash(self.name)
 
 
+class HPAData(pd.BaseModel):
+    min_replicas: int
+    max_replicas: int
+    current_replicas: int
```

Comment:
I could be wrong but I remember having an issue with this field being empty while the autoscaller is starting. It is manditory according to the docs Iv seen.
I would have this as optional too just to be safe so we dont have an exception in the rare case an autoscaller is starting up.